File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantsCreatePlantAdditivesRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantsCreatePlantAdditivesRequestItem {
    @JsonProperty("ActiveIngredients")
    public List<PlantsCreatePlantAdditivesRequestItemActiveIngredients> activeIngredients;
    @JsonProperty("ActualDate")
    public String actualDate;
    @JsonProperty("AdditiveType")
    public String additiveType;
    @JsonProperty("ApplicationDevice")
    public String applicationDevice;
    @JsonProperty("EpaRegistrationNumber")
    public String epaRegistrationNumber;
    @JsonProperty("PlantLabels")
    public List<String> plantLabels;
    @JsonProperty("ProductSupplier")
    public String productSupplier;
    @JsonProperty("ProductTradeName")
    public String productTradeName;
    @JsonProperty("TotalAmountApplied")
    public Integer totalAmountApplied;
    @JsonProperty("TotalAmountUnitOfMeasure")
    public String totalAmountUnitOfMeasure;
    public static class PlantsCreatePlantAdditivesRequestItemActiveIngredients {
    @JsonProperty("Name")
    public String name;
    @JsonProperty("Percentage")
    public Double percentage;
}
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsCreatePlantAdditivesRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsCreatePlantAdditivesRequestItem(
    @JsonProperty("activeIngredients")
    val activeIngredients: List<PlantsCreatePlantAdditivesRequestItemActiveIngredients>? = null,
    @JsonProperty("actualDate")
    val actualDate: String? = null,
    @JsonProperty("additiveType")
    val additiveType: String? = null,
    @JsonProperty("applicationDevice")
    val applicationDevice: String? = null,
    @JsonProperty("epaRegistrationNumber")
    val epaRegistrationNumber: String? = null,
    @JsonProperty("plantLabels")
    val plantLabels: String? = null,
    @JsonProperty("productSupplier")
    val productSupplier: String? = null,
    @JsonProperty("productTradeName")
    val productTradeName: String? = null,
    @JsonProperty("totalAmountApplied")
    val totalAmountApplied: Int? = null,
    @JsonProperty("totalAmountUnitOfMeasure")
    val totalAmountUnitOfMeasure: String? = null
)
